### ISSUE TITLE
IPC: cannot receive stdout 

### ISSUE THREAD

**Issue by lolBig** [Rokid/ShadowNode#171]
```js
child.fork('child.js', {
  silent: true,
})
```
如果child.js的stdout比较少可以收到，如果child.js替换成homebase/index.js，则收不到。

**Comment by yorkie** [comment]
@lolBig Could you please give the detailed case to reproduce this issue?

**Comment by yorkie** [comment]
And describe what the stdout is in more details, thanks :)

**Comment by lolBig** [comment]
```js
console.log('11111')
setTimeout(function() {}, 5011110)
```

**Comment by yorkie** [comment]
@legendecas please helps this.

**Comment by legendecas** [comment]
A shell pipe can simply reproduce this issue too.

**Comment by legendecas** [comment]
Caused by not flushed stdout stream. To be fixed next.

**Comment by yorkie** [comment]
Closed, this has been fixed at #182.
